pkg/splunk/enterprise/afwscheduler.go

```diff
@@ -343,6 +343,13 @@ func (downloadWorker *PipelineWorker) download(pplnPhase *PipelinePhase, s3Clien
 	err = s3ClientMgr.DownloadApp(remoteFile, localFile, appDeployInfo.ObjectHash)
 	if err != nil {
 		scopedLog.Error(err, "unable to download app", "appName", appName)
+
+		// remove the local file
+		err = os.RemoveAll(localFile)
```

Comment:
This is one of the bugs found as part of CSPL-1515 where if we are not able to download the app/s due to any reason, we would be left with stale 0 sized files on the operator pod. And in the next reconcile we would just check that the file is present on operator pod(without checking the size) and would move to podCopy and then to install phase. Then at the install phase we would fail to install the app/s as it is of size 0. Hence we need to remove the file on the operator pod.